src/rpiplatesrecognition/libs/plate_acquisition/__init__.py

```diff
@@ -25,15 +25,6 @@ def global_edge_projection(img: np.ndarray, parameters: ExtractionConfigParamete
 
     return result
 
-def global_edge_projection(img: np.ndarray, parameters: ExtractionConfigParameters):
-    if parameters.algorithm_choice == 1:
-        result = edge_projection_algorithm(img, parameters)
-    elif parameters.algorithm_choice == 2:
-        result = edge_projection_algorithm_v2(img, parameters)
-    else:
-        raise ValueError()
-
-    return result
 
```

Comment:
Don't delete this, i'm using this in admin panel

Reply:
there was fun duplicate